README.md

```diff
@@ -254,13 +254,7 @@ The site can now be accessed at [localhost:1234](http://localhost:1234).
   - Required packages:
     - `node` = 18
 
-In Debian/Ubuntu, install dependencies:
-
-```sh
-$ sudo apt update # Update the package list
-$ sudo apt install golang g++ libc-dev libheif-dev libdlib-dev libjpeg-dev libblas-dev libatlas-base-dev liblapack-dev # For API requirement
-$ sudo apt install reflex sqlite3 # For API optional tools
-```
+In Linux, please use [Docker Compose or Docker](https://github.com/googollee/photoview?tab=readme-ov-file#set-up-docker-development-environment).
```

Comment:
There are (were) contributors, who don't want/like to use Docker-based dev env. I guess that we can keep the old instructions for them in the Readme.
Same for lines 300-301

Reply:
It's conflicted with [your reply](https://github.com/photoview/photoview/issues/1031#issuecomment-2323202255). As we are not building on Debian, these packages may not be installed in the future, and we can't verify it easily.

Since we list dependencies above, developers know which packages to be installed in their env.

Reply:
Ok, I see your point.
Well, for the Docker-based deployments, we provide a fully integrated package (image + PROD compose + DEV compose + optional Makefile). However, on the other side, we still support the direct host deployment scenario, and for that scenario, we have to provide the list of dependencies and deployment steps. Currently, we have the list of supported distros and the migration of our image to Alpine should increase this list, not decrease.
This doesn't mean that we have to provide installation scripts and other easy-install stuff (unless we want to), but we need to provide the list of prerequisites and requirements to make Photoview work on currently supported systems.

In short: in addition to recommending users to use Docker and warn them about the risk of these instructions becoming outdated, please keep the direct installation instructions as well.

> As we are not building on Debian, these packages may not be installed in the future, and we can't verify it easily.

This risk is acceptable for the dev env. You can mention in the warning above, that devs have to be ready to resolve some dependency-related issues and are expected to contribute updates to the readme with the tested corrections.
However, after some time we might see the same type of issues with our PROD host deployment dependency inconsistencies. Since both of us don't use host deployments, the best we can do is to react to users' reports and try to fix those issues. I don't think that this risk should stop us from improving the product.

Reply:
Another reason of removing those Debian commands is that they are already broken in Debian stable and Ubuntu stable, because the library of go-heif `v1.17` won't be compiled with the `libheif` from the distribution. I can keep those instructions in README, but I guess it just confuses people.

Reply:
Added those commands back.

Reply:
> I can keep those instructions in README, but I guess it just confuses people.

Ok, let's see... At least, this will be the last working configuration, so devs could take it as a starting point and fix inconsistencies - that is easier than inventing it from scratch, I guess